omod/src/main/webapp/fragments/header.gsp

```diff
@@ -114,7 +116,7 @@
         <li class="change-location">
```

Comment:
I started with something like 
`<span id="location-uuid" calss="hidden">${ sessionContext.sessionLocation.uuid }</span>`
There where two problems:
1. The uuid was not changing after selecting new location
2. In the UI Tests I couldn't get the element that was hidden